cmd/auth/main.go

```diff
@@ -258,7 +264,34 @@ func initSchema(ctx context.Context, client *authzed.ClientWithExperimental, sch
 	return nil
 }
 
-func newService(db *sqlx.DB, tracer trace.Tracer, cfg config, dbConfig pgclient.Config, logger *slog.Logger, spicedbClient *authzed.ClientWithExperimental, cacheClient *redis.Client, keyDuration time.Duration, keyManager auth.KeyManager, idProvider supermq.IDProvider) (auth.Service, error) {
+func validateKeyConfig(isSymmetric bool, cfg config) error {
+	if isSymmetric {
+		if cfg.SecretKey == "secret" {
+			return fmt.Errorf("default secret key is insecure - please set SMQ_AUTH_SECRET_KEY environment variable")
+		}
+		return nil
+	}
+
+	// Validate active key path
+	_, err := os.Stat(cfg.ActiveKeyPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("active key file not found: %s - please set SMQ_AUTH_KEYS_ACTIVE_KEY_PATH", cfg.ActiveKeyPath)
+		}
+		return fmt.Errorf("failed to access active key file: %w", err)
+	}
+
+	// Retiring key is optional - only validate if path is provided
+	if cfg.RetiringKeyPath != "" {
+		if _, err := os.Stat(cfg.RetiringKeyPath); err != nil {
```

Comment:
Why don't here not checking `os.IsNotExist` ?

Reply:
This should not return an error, fixed.